themes/digital.gov/layouts/shortcodes/img.html

```diff
@@ -1,3 +1,22 @@
+{{/* =======
+
+  Displays an image with optional metadata
+  See wiki for more details https://github.com/GSA/digitalgov.gov/wiki/Hugo-shortcodes
+  TODO: finish documenting the other properties of this component
+
+  align - set the alignment for the image to be left (default if left blank), center, or right aligned.
+  inline - displays multiples images on the same line
+
+  @example for right aligned image
+  {{< img src="image-1" align="right" alt="UX Summit Image"
+>}}
```

Comment:
@lcummings12 I added some documentation for the new options and described what this component does.